qa/automation/ScoringPortal.Tests.Scenarios/3.Sign_in_with_GoogleAcc/1. sign_in.feature

```diff
@@ -0,0 +1,36 @@
+Feature: 3.1 Sign in with Google Account
+
+Background:
+	Given user turn on VPN with necessary config
+	And go to https://qa.ddso-spot.quantori.com/
+
+Scenario: 3.1_01 User has Google account
+	Given User has a saved Google account
+	When User press the button "Sign in with Google"
+	And User select necessary Google account 
+	Then the page with Quantori Scoring is opened
+
+Scenario: 3.1_02 User doesn't have a saved Google account
```

Comment:
This scenario can be split into some more scenarios. There are at two pages with inputs in this scenario, and both of them can be tested using testing techniques we learned (EP, DT or STT, for example). All inputs have different error messages for different data.

Reply:
Discussion in slack approved my opinion that we shouldn't test google sign-in form.